index.html

```diff
@@ -311,31 +319,36 @@ <h5>
     <!-- Spacing between the tabs and lower links and footer-->
     <br>
 
-
-    <div class="conatiner lower-navbar">
+    <div class="container lower-navbar">
       <div class="row">
         <div class="col-6" style="padding-top: 8px; padding-left: 0px; margin-left: 0px;">
           <h5>
             <a target="_blank" rel="noopener noreferrer" href="https://github.com/KnossosNET/Knossos.NET/releases">
-              All Releases  
+              All Releases
             </a>
-            
-            &ensp;<b>|</b>&ensp;        
-            
+
+            &ensp;<b>|</b>&ensp;
+
             <a target="_blank" rel="noopener noreferrer" href="https://www.hard-light.net">
-              Hard-Light Productions 
+              Hard Light Productions
             </a>
 
-            &ensp;<b>|</b>&ensp; 
-            
+            &ensp;<b>|</b>&ensp;
+
             <a target="_blank" rel="noopener noreferrer" href="https://discord.gg/cyhMBhMHzK">
-              Hard-Light Discord 
+              Hard Light Discord
             </a>
 
             &ensp;<b>|</b>&ensp;
-            
+
+            <a target="_blank" rel="noopener noreferrer" href="https://scp.indiegames.us">
+              FreeSpace Open
+            </a>
+
+            &ensp;<b>|</b>&ensp;
+
             <a target="_blank" rel="noopener noreferrer" href="https://pxo.nottheeye.com">
-              PXO
```

Comment:
If we're not adding explanations for the other options, then we shouldn't add one here.  Hard Light as a name is just as opaque to a newcomer as PXO.

Reply:
Well, ok.  Removed.